xblock/django/request.py

```diff
@@ -76,9 +80,15 @@ def querydict_to_multidict(query_dict, wrap=None):
 
     """
     wrap = wrap or (lambda val: val)
+
+    if future.utils.PY2:
+        item_lists = query_dict.iterlists()
+    else:
+        item_lists = query_dict.lists()
```

Comment:
For expected-short lists, wouldn't it be better to just use the common method name that makes a list in Py2?